src/main/resources/i18n/Enums_en.properties

```diff
@@ -170,9 +170,9 @@ public.UploadStatus.Validating=Validating
 public.UploadType.AccessionCSV=Accession CSV
 public.UploadType.SeedlingBatchCSV=Seedling Batch CSV
 public.UploadType.SpeciesCSV=Species CSV
-public.WoodDensityLevel.FamilyLevel=Family level
-public.WoodDensityLevel.GenusLevel=Genus level
-public.WoodDensityLevel.SpeciesLevel=Species level
+public.WoodDensityLevel.Family=Family
+public.WoodDensityLevel.Genus=Genus
+public.WoodDensityLevel.Species=Species
```

Comment:
Do we still want "level" to appear in the human-readable values in search results and such? Totally fine if we don't need it there, of course.

Reply:
I will double check the fields in the PRD with design/product

Reply:
Confirmed, do not want "level" in the value label.